### PR TITLE
Add shared video smart mike mutes unmutes

### PR DESCRIPTION
**Pull request by yanas**

### CODE REVIEW

File: modules/UI/shared_video/SharedVideo.js [hunk 1]
```diff
@@ -421,20 +438,63 @@ export default class SharedVideoManager {
...
+    }
+
+    /**
+     * Mutes / unmutes the player.
+     * @param mute true to mute the shared video, false - otherwise.
+     */
+    mutePlayer(mute) {
+        if (!this.player.isMuted() && mute) {
+            this.player.mute();
+            this.smartUnmute();
```
Comment (damencho) [inline modules/UI/shared_video/SharedVideo.js]: I was trying to not use the player mute, cause it brings another complexity. As the player can be muted and put with volume to 0. So everywhere we were using just volume 0. Just on the checks where we need to send info for player state we do: this.player.isMuted() ? 0 : this.player.getVolume(). If its muted we just send volume 0, which produces muted player.

**Comment by damencho** [comment]
Something in general bothers me: 
incoming presence -> processAttributes -> mutePlayer -> updateCheck -> send presence, and as we receive our own presences and process them for the owner of the shared video we can start sending presences in a loop.

File: modules/UI/shared_video/SharedVideo.js (hunk 1 above)
Comment (damencho) [inline modules/UI/shared_video/SharedVideo.js]: Is this necessary, can we stick with volume stuff.

Comment (yanas) [inline modules/UI/shared_video/SharedVideo.js]: As we discussed a little bit off-site, we need this in order to keep the volume untouched when the user mutes the video and then restore the volume once it's unmuted. I think I've taken care of the handling and Damyan agreed that it looks good for now.

**Comment by yanas** [comment]
I agree completely that the level of complexity we have added around audio and shared video mutes doesn't fit very well. It's a little dispersed and could be source of bugs. I'll try to simplify and optimise the code.

**Comment by ezekia** [comment]
okey

**Comment by ezekia** [comment]
I was just wanting to configure my own Jitsi meet of my own. (My own server
of Jitsi ) but when I try I get confused how to set DNS and server itself.
any Help plse.....

**Comment by ezekia** [comment]
I was trying to use the Sample server configured for Conference and my link
is https://meet.jit.si/EnormousPigsMeetIntensely
i dont know how another do join. can try my link and if you can connect to
me?

**Pull request merged**